admin/class-manage-fonts.php

```diff
@@ -229,6 +229,51 @@ function google_fonts_admin_page() {
 	<?php
 	}
 
+    function delete_font_asset ( $font_face ) {
+        // if the font asset is a theme asset, delete it
+        $theme_folder = get_stylesheet_directory();
+        $font_path = pathinfo( $font_face['src'][0] );
+        $font_dir = str_replace("file:./", "", $font_path['dirname']);
+
+        $font_asset_path = $theme_folder . DIRECTORY_SEPARATOR . $font_dir . DIRECTORY_SEPARATOR . $font_path['basename'];
+
+        if ( ! is_writable( $theme_folder . DIRECTORY_SEPARATOR . $font_dir ) ) {
+            return add_action( 'admin_notices', [ $this, 'admin_notice_font_asset_removal_error' ] );
+
+            if ( file_exists( $font_asset_path ) ) {
```

Comment:
I think this needs to be moved outside of the current `if` statement.

Reply:
Thanks for catching this! It's fixed now.